packages/amplify-category-api/package.json

```diff
@@ -120,6 +120,7 @@
   },
   "devDependencies": {
     "@aws-cdk/assertions": "~1.124.0",
+    "@types/node": "^18.7.13",
```

Comment:
I suspect if we should use `18.x` here since we do not yet support that node version. Can you please change this to `^12.12.6` which is the version we use across [packages in our repo](https://github.com/aws-amplify/amplify-category-api/blob/d7bf74dae3b8e05ef4f31a902f4e7ea14ca4e127/packages/amplify-graphql-transformer-core/package.json#L71)? Thanks.

Reply:
Hey @phani-srikar totally missed this response, and while a valid point should this instead be bumped across packages to Node 14?

Reply:
this package version appears to coincide with a typescript version, which I've also added as a devDependency https://www.npmjs.com/package/@types/node